bridge/.solcover.js

```diff
@@ -1,7 +1,13 @@
 module.exports = {
-    skipFiles: ['test','zeppelin', 'previous'],
-    providerOptions: {
-      network_id: 5888,
-      //vmErrorsOnRPCResponse: false
-    }
-  };
\ No newline at end of file
+  skipFiles: [
+    'test',
+    'zeppelin',
+    'Federation/FederationV1.sol',
+    'SideTokenFactory/SideTokenFactoryV1.sol',
+    'SideToken/SideTokenV1.sol'
```

Comment:
I think the coverage is low because you aren not skiping Utils

Reply:
We may also want to skip the proxies

Reply:
done at https://github.com/rsksmart/tokenbridge/pull/267/commits/2e29b465e2dc78ce337313159faedf3b55aaff43